packages/vue/src/components/molecules/SfModal/SfModal.html

```diff
@@ -0,0 +1,17 @@
+<section class="sf-modal" :class="{ 'sf-modal--visible': visible }">
+  <SfOverlay class="sf-modal__overlay" :transition="transitionOverlay" v-show="visible" v-if="overlay" @click="checkPersistence"></SfOverlay>
+  <transition :name="transitionModal">
+    <div class="sf-modal__container" v-show="visible">
+      <button class="sf-modal__cross" @click="toggle" v-if="cross">
+        <!--@slot Use this slot to place content inside the close button.-->
+        <slot name="close">
+          <img src="/assets/close.svg" class="sf-circle-icon__icon" alt="">
```

Comment:
```suggestion
          <img src="/assets/close.svg" class="sf-modal__icon" alt="">
```
you dont have block sf-circle-icon or I can't find it :(